bluebottle/bluebottle_dashboard/templates/exportdb/base.html

```diff
@@ -1,76 +0,0 @@
-
-{% block extrahead %}{{ block.super }}
-<script type="text/javascript" src="{% static 'admin/js/vendor/jquery/jquery.js' %}"></script>
-<script type="text/javascript" src="{% static 'admin/js/jquery.init.js' %}"></script>
-<script type="text/javascript" src="{% static 'admin/js/core.js' %}"></script>
-<script type="text/javascript" src="{% url 'admin:jsi18n' %}"></script>
-{{ form.media }}
-{% endblock %}
-
-
-{% block extrastyle %}{{ block.super }}<link rel="stylesheet" type="text/css" href="{% static "admin/css/forms.css" %}" />{% endblock %}
-
-{% block content %}
-<div id="content-main">
-    <form action="." method="post">
-        <div>
-
-            {% if form.errors %}
-                <p class="errornote">
-                {% if form.errors|length == 1 %}{% trans "Please correct the error below." %}{% else %}{% trans "Please correct the errors below." %}{% endif %}
-                </p>
-                {{ form.non_field_errors }}
-            {% endif %}
-
-            {% if form.fields %}
-
-                <fieldset class="module aligned">
-                    <h2>{% trans "Extra fields" %}</h2>
-                    {% for field in form %}
-                        <div class="form-row{% if field.errors %} errors{% endif %}">
-                            <div{% if field.is_checkbox %} class="checkbox-row"{% endif %}>
-                                {{ field.errors }}
-                                {% if field.is_checkbox %}
-                                    {{ field }}{{ field.label_tag }}
-                                {% else %}
-                                    {{ field.label_tag }}
-                                    {% if field.is_readonly %}
-                                        <p>{{ field.contents|linebreaksbr }}</p>
-                                    {% else %}
-                                        {{ field }}
-                                    {% endif %}
-                                {% endif %}
-                                {% if field.help_text %}
-                                    <p class="help">{{ field.help_text|safe }}</p>
-                                {% endif %}
-                            </div>
-                        </div>
-                    {% endfor %}
-                </fieldset>
-
-            {% endif %}
-
-            <fieldset class="module aligned">
-                <h2>{% trans "Confirm export" %}</h2>
-
-                <div class="form-row">
-                    {% trans "The following object types will be exported" %}
-                    <ul>
-                        {% for model in models %}
-                            <li>{{ model }}</li>
-                        {% endfor %}
-                    </ul>
-                </div>
-            </fieldset>
-
-        </div>
-
-
-        <div class="submit-row">
-            <input type="submit" value="{% trans "Confirm" %}" class="default" name="_save">
-        </div>
-    </form>
-</div>
-{% endblock %}
```

Comment:
Jep. We were overriden them because the templates were not django 1.9 compatible. Now they are.